docs/timelines/running-and-editing-timelines.md

```diff
@@ -0,0 +1,138 @@
+
+- Left arrow - moves selection to the previous trigger point on the track.
+- Right arrow - moves selection to the next trigger point on the track.
+
+If nothing is selected, the right arrow will select the first trigger on the track and the left arrow will select the last.
+
+### Using the wheels to edit triggers
+
+When you have one or more trigger points selected, the wheels control the time, level and fade of those triggers (some types of trigger do not have all these parameters).
+
+- Wheel A sets the time.
+- Wheel B sets the level.
+- Wheel C sets the fade.
+
+You can change multiple triggers at the same time by selecting them all and using the wheels.
+
+### Using the overview bar
+
+The **Overview Bar** at the bottom of the Timeline View gives you a miniature view of all the events in the track.
+
+![Timeline overview bar](/docs/images/Timeline-Overview-Bar.png)
+
+- The handles at the left and right of the Overview Bar set the start and end points of the main view above. This gives you an easy way to zoom in on a particular area.
+
+- When you are zoomed in you can drag on the zoomed area in the bar to move the main view (the same as using the Hand tool and dragging on the main view).
+
+- The edit cursor is shown in grey.
+
+- The current live time is shown in blue.
+
+### Deleting triggers
+
+You can delete trigger blocks by pressing \<Delete\> then tapping on the block you want to delete, or drawing a marquee selection box over the triggers to be deleted.
+
+You can also delete tracks or delete rows from tracks using this method.
+
+### Setting track legends and halo colours on tracks
+
+To help you identify different tracks, you can set halo colours. This colour outlines the track and also sets the colours of the trigger blocks which makes them easier to see in the overview bar.
+In addition you can set different colours for the playback actions within each track as shown in Track 2 in the picture below.
+
+![Timeline halo](/docs/images/Timeline-Halo.png)
+
+1. Press \[Set Legend\] at the top level main menu.
+
+2. Tap on the Track legend or the playback row in the Timeline View.
+
+3. Change \[Legend\] to set the track legend or press \[Halo\].
+
+4. Select a halo colour from the colour picker.
+
+- You can remove a halo colour by pressing \[Remove Halo\] at step 4.
+
+- You can also set legends on markers using the \[Set Legend\] button.
+
+- You can set a halo colour for a **Wait for Go** trigger by tapping on it at step 2.
+
+### Snap options
+
+Using the context menu option \{Snap Options\} you can configure whether new triggers will
+snap to existing objects when you enter them by clicking on the Timeline View.
+The snap options are
+- Snap To Triggers
+- Snap To Markers
+- Snap To Cursor.
+
+
+### Collapsing track view
+
+If you have a lot of tracks, you can collapse them to a single line to show more tracks on the screen, by clicking the arrow button to the left of the track name.
```

Comment:
Might be good to describe what the semi-collapsed state does.

Perhaps deduplicate from `docs/timeline.md`?

Reply:
Semi-collapsed was not in the version I was testing... I see it now, what does it actually do? It seems to sometimes combine playback rows in the track and sometimes not but I can't work out when each happens.

Reply:
So it tries to lay out the playback level groups in the track using the fewest number of rows possible; so
```
[---A--]
      [---B---]
             [---C---]
```
would instead be laid out
```
[---A--] [---C---]
      [---B---]
```

In the uncollapsed state at the top, each playback has its own row, in the collapsed state at the bottom they share as can be made to fit.